### PR TITLE
VoteAdmin with correct Role orgs

### LINKED ISSUE #189: All voting admins for the same church should be able to access, administrate polling events of other voting admins for the same church
**User Story**
e.g. if Lene is voting admin for Oslo and Morten is also voting admin for Oslo. Than they both should be able to see & administrate the voting events the other created even if they are not in the defined user group of the voting event

**Acceptance Criteria**
1. Administrators of the same church can access & administrate the poll events of the other administrators of the same church

### PR DESCRIPTION
**Pull request by StevenMalaihollo**
Closes #189

### CODE REVIEW

**Review by JakubC-projects — COMMENTED** [review]
This actually fails 8 tests. I think it's because the test data wasn't updated, so I guess we should update that before releasing.

File: server/src/domain/User.ts [hunk 1]
```diff
@@ -1,12 +1,29 @@
...
+    email: string;
+    roles: Array<UserRole>;
+    displayName: string;
+    age: number;
+    cellPhone?: any;
+}
+
+export type UserRole = Role & {
+    organisationID: Array<number>;
+    org_id: Array<string>;
```
Comment (JakubC-projects) [inline server/src/domain/User.ts:18]: Why we have both organisationID and org_id? I see that org_id is never used, so I think it should be removed and organisationID would then be renamed to orgID or something (it feels wierd to use the '_' notation here).

File: server/src/permissions/appAbility.ts [hunk 2]
```diff
@@ -53,21 +53,22 @@ const rolePermissions: Record<string, DefinePermissions> = {
...
+        const votingAdminFor = user.roles.filter((r:UserRole) => r.enumName == 'VotingAdmin')[0].organisationID;
         can('create','poll');
         can('update', 'poll');
         can('remove', 'poll');
         can('patch', 'poll');
 
-        can('patch', 'polling-event', { 'participantFilter.org': userChurchID });
-        can('update', 'polling-event', { 'participantFilter.org': userChurchID });
-        can('create','polling-event', { 'participantFilter.org': userChurchID });
+        can('patch', 'polling-event', { 'participantFilter.org': { $in: votingAdminFor}} as any);
```
Comment (JakubC-projects) [inline server/src/permissions/appAbility.ts:63]: Why do we use $in if votingAdminFor is just a number?

Comment (StevenMalaihollo) [reply server/src/permissions/appAbility.ts:63]: votingAdminFor is an array of numbers like defined in the types

File: server/src/domain/User.ts (hunk 1 above)
Comment (StevenMalaihollo) [reply server/src/domain/User.ts:18]: organisationID is our external ids array right. And org_id is the internal arango db id.

We shouldn't use the org_id in vote (external system) but we have this array on the UserRole so we might aswell have it in the types right.

File: server/src/permissions/appAbility.ts (hunk 2 above)
Comment (JakubC-projects) [reply server/src/permissions/appAbility.ts:63]: Oh, sorry It really is, but I guess we should rename the organisationID to something that would indicate that it's an array

File: server/src/domain/User.ts (hunk 1 above)
Comment (JakubC-projects) [reply server/src/domain/User.ts:18]: But we could remove that array. It doesn't have to exist, does it? For  me it doesn't feel right to store unnecessary data especially that it has a similar name to used data

Comment (StevenMalaihollo) [reply server/src/domain/User.ts:18]: No it is necassary, thats the whol point of this PR: You can be a votingAdmin for multiple churches

Comment (JakubC-projects) [reply server/src/domain/User.ts:18]: I understand now that organisationID is an array and it is necessary but the org_id isn't, am I right?

Comment (StevenMalaihollo) [reply server/src/domain/User.ts:18]: Yes

**Review by JakubC-projects — APPROVED** [review]

**Pull request merged**